### PR TITLE
docs: add reference for BCP 47

### LINKED ISSUE #571: docs: Add notes on language metadata and keyboard fonts to Package Tutorial / Help
The Package Tutorial, Package metadata reference and Package Editor context help do not include sufficient information on what a BCP 47 tag is. Furthermore, the tutorial is missing any information on completing keyboards/lexical model metadata.

Relevant Keyman Developer documentation pages that need updates:
* [Package Tutorial](https://help.keyman.com/developer/15.0/guides/distribute/tutorial/)
* [Package Editor context help](https://help.keyman.com/developer/15.0/context/package-editor#keyboards)
* [Package metadata reference](https://help.keyman.com/developer/current-version/reference/file-types/metadata#obj-language)
* [&ethnologueCode store](https://help.keyman.com/developer/language/reference/ethnologuecode)? (deprecated)

Possible good example:
* [Lexical Model Tutorial](https://help.keyman.com/developer/15.0/guides/lexical-models/tutorial/step-2#toc--determine-your-language-s-bcp-47-language-tag-)

Ref: https://community.software.sil.org/t/set-new-bcp-47-code-to-make-android-keyboard/6442

### PR DESCRIPTION
**Pull request by mcdurdin**
Fixes #571.

Also updates three files to markdown, but leaves remaining three as PHP for now -- they only had minor link changes.

### CODE REVIEW

File: developer/16.0/reference/bcp-47.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+* facilitating searches for keyboards and lexical models on the Keyman site,
+  through a [.keyboard_info file](/developer/cloud/keyboard_info)
+
+A BCP 47 language tag is made up of multiple subtags. There are many possible
+subtags, but only three types are currently used in most places in Keyman
+Developer:
+
+* [language subtag](#the-language-subtag)
+* [script subtag](#the-script-subtag)
+* [region subtag](#the-region-subtag)
```
Comment (darcywong00) [inline developer/16.0/reference/bcp-47.md:22]: hmm, these anchors weren't working for me

**Review by darcywong00 — DISMISSED** [review]
lgtm

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by mcdurdin** [comment]
> oops - link-checker flagged a broken link

Per our review process, you don't need to cancel an approval just because of a build failure or CI check failure. It's the responsibility of the PR author to ensure that everything passes, and to ask for re-review if they need to make significant changes.

**Pull request merged**